docs/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+
+## 1.0.0b1
+
+Feature:
+* Get configs via Airflow Configuration manager [#503](https://github.com/astronomer/astro-sdk/issues/503)
+* Optimize loading file directly from GCS to Bigquery via load_file operator [#429](https://github.com/astronomer/astro-sdk/issues/429)
+* Optimize loading file directly from S3 to Bigquery via load_file operator [#429](https://github.com/astronomer/astro-sdk/issues/429)
+* Optimize loading file directly from local to bigquery via load_file operator [#429](https://github.com/astronomer/astro-sdk/issues/429)
+* Optimize performance loading file to postgres via load_file operator [#428](https://github.com/astronomer/astro-sdk/issues/428)
+* Optimize loading file directly from GCS to snowflake via load_file operator [#430](https://github.com/astronomer/astro-sdk/issues/430)
+* Optimize loading file directly from S3 to snowflake via load_file operator [#430](https://github.com/astronomer/astro-sdk/issues/430)
+* Add user-facing doc-site using Sphinx [#472](https://github.com/astronomer/astro-sdk/issues/472)
+
+Internals:
+* Fix sphinx docs sidebar [#472](https://github.com/astronomer/astro-sdk/issues/472)
+* Add CI job to check for dead links
+
+Enhancement:
+* Define datasets to be used in the benchmark [#433](https://github.com/astronomer/astro-sdk/issues/433)
+* Define benchmark worker node configuration for benchmark [#435](https://github.com/astronomer/astro-sdk/issues/435)
+* Change benchmark tool to run in the GKE [#432](https://github.com/astronomer/astro-sdk/issues/432)
+* Add benchmark result for load_file to snowflake using Astro Python SDK [#438](https://github.com/astronomer/astro-sdk/issues/438)
+* Add postgres benchmarking result [#431](https://github.com/astronomer/astro-sdk/issues/431)
+* Fix issue in default schema selection [#503](https://github.com/astronomer/astro-sdk/issues/503)
+* Fix missing airflow's task terminal states to CleanUp Operator [#525](https://github.com/astronomer/astro-sdk/issues/525)
+* Allow running tests on PRs from forks + label [#179](https://github.com/astronomer/astro-sdk/issues/179)
```

Comment:
This is meaningful to the development community of the Astro SDK, so it should be placed within internals, perhaps.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/576/commits/88b0e130c53839808f7fa7be3df528526f57e8ab)